src/Microsoft.AspNetCore.Mvc.Razor/TagHelpers/ITagHelperComponentPropertyActivator.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.TagHelpers
+{
+    /// <summary>
+    /// Provides methods to activate properties of <see cref="ITagHelperComponent"/>s.
+    /// </summary>
+    public interface ITagHelperComponentPropertyActivator
+    {
+        /// <summary>
+        /// Activates properties of the <see cref="ITagHelperComponent"/>.
```

Comment:
Use a param ref for yout `tagHelperComponent` instead of `<see cref="ITagHelperComponent"/>`